src/cc_catalog_airflow/Dockerfile

```diff
@@ -16,7 +16,7 @@ ADD ./requirements.txt ${AIRFLOW_HOME}
 ADD ./wait_for_db.py ${AIRFLOW_HOME}
```

Comment:
ADD ./requirements.txt needs to be changed to requirements_dev.txt

Reply:
Yeah I am closing this PR and have done this in https://github.com/creativecommons/cccatalog/pull/365 only.